config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -50,7 +50,7 @@ periodics:
       - --
       - --cluster=gce-golang
       - --env=CL2_ENABLE_PVS=false
-      - --env=CL2_LOAD_TEST_THROUGHPUT=40
+      - --env=CL2_LOAD_TEST_THROUGHPUT=50
```

Comment:
50 won't work, because controllers have lower throughput than they need (even on avg).

Reply:
In other words - you would need to bump QPSes to make it work.

Reply:
Thanks for catching, done.